packages/slidev/node/plugins/markdown.ts

```diff
@@ -97,6 +97,13 @@ export async function createMarkdownPlugin(
 
 export function transformMarkdownMonaco(md: string) {
   // transform monaco
+  md = md.replace(/^```(\w+?)\s*{monaco}\s*?({.*?})?\s*?\n([\s\S]+?)^~~~\s*?\n([\s\S]+?)^```/mg, (full, lang = 'ts', options = '{}', code: string, diff: string) => {
```

Comment:
```suggestion
  md = md.replace(/^```(\w+?)\s*{monaco-diff}\s*?({.*?})?\s*?\n([\s\S]+?)^~~~\s*?\n([\s\S]+?)^```/mg, (full, lang = 'ts', options = '{}', code: string, diff: string) => {
```

I am thinking of integrating retypewriter and other tools later with similar syntax (multiple-step blocks). I think we might be better to be explicit here.

Reply:
What do you think about the separator `~~~` inside the code block ?

Reply:
I was thinking to introduce a syntax for that indeed and I think I end up with the same as your idea of using `~~~`